packages/web-app-files/src/components/SideBar/Shares/Collaborators/RoleDropdown.vue

```diff
@@ -293,7 +290,9 @@ export default defineComponent({
     },
 
     cancelCustomPermissions() {
-      this.customPermissions = this.existingPermissions
+      this.customPermissions = this.existingPermissions.length
```

Comment:
FYI this is the important change here. The rest is just refactoring/cleaning outdated stuff,.

Reply:
👍